cfme/tests/services/test_service_catalogs.py

```diff
@@ -190,7 +189,9 @@ def test_no_template_catalog_item(provider_crud, provider_type, provisioning,
     catalog_item = CatalogItem(item_type=catalog_item_type, name=item_name,
                   description="my catalog", display_in=True, catalog=catalog, dialog=dialog)
     catalog_item.create()
-    flash.assert_message_match("'Catalog/Name' is required")
+    flash.assert_message_match(version.pick({version.LOWEST: "'Catalog/Name' is required",
+                                             "5.4": "Source_id must have valid template",
+                                             version.LATEST: "Source_id must have valid template"}))
```

Comment:
You don't have to supply the `LATEST` version, it knows it has to use the latest one, `"5.4"` in this case.